src/check-binary-license

```diff
@@ -88,32 +88,34 @@ done
 
 # check pulsar sql jars
 JARS=$(tar -tf $TARBALL | grep '\.jar' | grep 'lib/presto/' | grep -v pulsar-client | grep -v bouncy-castle-bc | grep -v pulsar-metadata | grep -v 'managed-ledger' | grep -v  'pulsar-client-admin' | grep -v  'pulsar-client-api' | grep -v 'pulsar-functions-api' | grep -v 'pulsar-presto-connector-original' | grep -v 'pulsar-presto-distribution' | grep -v 'pulsar-common' | grep -v 'pulsar-functions-proto' | grep -v 'pulsar-functions-utils' | grep -v 'pulsar-io-core' | grep -v 'pulsar-transaction-common' | grep -v 'pulsar-package-core' | sed 's!.*/!!' | sort)
-LICENSEPATH=$(tar -tf $TARBALL  | awk '/^[^\/]*\/lib\/presto\/LICENSE/')
-LICENSE=$(tar -O -xf $TARBALL "$LICENSEPATH")
-LICENSEJARS=$(echo "$LICENSE" | sed -nE 's!.* (.*\.jar).*!\1!gp')
-
-
-for J in $JARS; do
-    echo $J | grep -q "org.apache.pulsar"
-    if [ $? == 0 ]; then
-        continue
-    fi
-
-    echo "$LICENSE" | grep -q $J
-    if [ $? != 0 ]; then
-        echo $J unaccounted for in lib/presto/LICENSE
-        EXIT=1
-    fi
-done
-
-# Check all jars mentioned in LICENSE are bundled
-for J in $LICENSEJARS; do
-    echo "$JARS" | grep -q $J
-    if [ $? != 0 ]; then
-        echo $J mentioned in lib/presto/LICENSE, but not bundled
-        EXIT=2
-    fi
-done
+if [ -n "$JARS" ]; then
```

Comment:
this is pretty dangerous, if there is a problem with the build we are skipping the test

Reply:
it's intentional. the script has been modified to work in the case where Pulsar SQL is not part of the distribution. that's a preparation for PIP-62 . Pulsar SQL will be built only for the "system tests" which require Pulsar SQL to be part of the built docker container. The binary check could be done there as well to validate the Pulsar SQL libraries.